tests/validation/test_validate.py

```diff
@@ -183,6 +183,55 @@ def test_validates_geojson_with_tuple_coordinates(self) -> None:
         # Should not raise.
         item.validate()
 
+    @pytest.mark.vcr()
```

Comment:
nice test! If you need vcr then there is probably a cassette file that you need to check in as well. In that case CI will fail. If CI passes then I think you don't need to have `pytest.mark.vcr` here.

Reply:
I am not too familiar with VCR. I assume I need the same references that are contained in `data-files/item/sample-item.json` ? I simply reused that item from another test case.

Reply:
I took a look and it seems like this decorator is good enough. Don't need a new cassette as well.